pkgs/os-specific/linux/zfs/generic.nix

```diff
@@ -199,7 +198,13 @@ let
     outputs = [ "out" ] ++ optionals buildUser [ "dev" ];
 
     passthru = {
-      inherit enableMail latestCompatibleLinuxPackages kernelModuleAttribute;
+      inherit enableMail kernelModuleAttribute;
+      latestCompatibleLinuxPackages = lib.pipe linuxKernel.packages [
+        builtins.attrValues
+        (builtins.filter (kPkgs: (builtins.tryEval kPkgs).success && kPkgs ? kernel && kPkgs.kernel.passthru.isVanilla && kernelCompatible kPkgs.kernel))
```

Comment:
For my learning: Is there a particular difference between using `kPkgs.kernel.passthru.isVanilla` and `kPkgs.kernel.isVanilla`?

Reply:
There is. If you are using ``placeholder "out"``, then using pkg.value it will get the ``out`` of that package. If you are using pkg.passthru.value then you get the ``out`` of the current package.

So using pkg.passthru.value could be wrong in some cases.

Reply:
🤔 I thought it was like that but apparently not. Not sure how I got onto that.